csrc/codegen.cpp

```diff
@@ -1001,6 +1051,50 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
   }
 
   void handle(const TernaryOp* top) final {
+    // Get vectorization information
```

Comment:
Can you add some comments about the expectation? IIUC, only `in2` is allowed to be vectorized, but technically speaking, it should be possible to have vectorized loads in both `in2` and `in3`, right? Not sure if it's worthwhile to allow that as well, although the required change seems minimal.

Reply:
Yes we can have in2 / in3 as TensorViews, I'm trying to add that since @zasdfgbnm mentioned about having a `where` test.